src/Writing/OpenAPISpecWriter.php

```diff
@@ -241,6 +233,9 @@ protected function generateEndpointRequestBodySpec(OutputEndpointData $endpoint)
                 $schema['properties'][$name] = $fieldData;
             }
 
+            if (array_key_exists('properties', $schema)) {
```

Comment:
`'properties'` gets erased when an array is encountered, so we need to check if it's still there